src/Containers/Search/Search.js

```diff
@@ -0,0 +1,53 @@
+import React, { useContext } from 'react';
+import { ProviderContext } from '../../contexts/ProviderContext';
+import { useLocation } from 'react-router-dom';
+import SearchByInsurance from '../../Components/SearchByInsurance/SearchByInsurance'
+import './_search.scss';
+
+
+
+const Search = () => {
+  const { doctorList,
+    setDoctorList,
+    filteredDoctorList,
+    setFilteredDoctorList,
+    mentalHealthProsList,
+    setMentalHealthProsList,
+    filteredMentalHealthProsList,
+    setFilteredMentalHealthProsList } = useContext(ProviderContext)
+console.log({ doctorList,
+  setDoctorList,
+  filteredDoctorList,
+  setFilteredDoctorList,
+  mentalHealthProsList,
+  setMentalHealthProsList,
+  filteredMentalHealthProsList,
+  setFilteredMentalHealthProsList })
+  const location = useLocation()
+  const pathname = location.pathname
+  let providerList;
+  let setFilteredProviderList;
+
+  if(pathname === '/doctors' ) {
+
+    providerList = doctorList;
+
+    setFilteredProviderList = setFilteredDoctorList
+
+  } else if (pathname === '/mental_health_professionals') {
+    console.log({mentalHealthProsList, setFilteredMentalHealthProsList})
```

Comment:
We'll probably want to get rid of this console log

Reply:
Agreed.